### ISSUE TITLE
Folder browse just shows Desktop

### ISSUE THREAD

**Issue by jcmolero** [layday/instawow#124]
I want to add the profile for the first addon set, but picking the folder just shows the desktop folder, nothing more, tried with the 2.0.1 and same result

**Comment by layday** [comment]
Could you take a screenshot? I'm not sure what just shows the desktop means.

**Comment by triat** [comment]
I had the same issue and if you click on your "desktop" this will show the path. In my case, the path was  "C:\bla\OneDrive\bla"

The important part here is the "\OneDrive\". I realized that the system can't show the whole folder hierarchy as my "Desktop" is, in fact, a link to the OneDrive folder.

One solution (in my case I did not wanted to have drive setup, one stone two birds) is to [remove the backup of the desktop folder on OneDrive](https://support.microsoft.com/en-us/office/turn-off-disable-or-uninstall-onedrive-f32a17ce-3336-40fe-9c38-6efb09f944b0)

Then, when you start instawow, this should show the whole folder hierarchy.
